### ISSUE TITLE
🐞 Fate dice doesn't work/ is replaced by a different dice

### ISSUE THREAD

**Issue by elvati** [javalent/dice-roller#361]
### Check for existing bug reports before submitting.

- [X] I searched for existing Bug Reports and found no similar reports.

### Expected Behavior

Fate rolls (for example, **2dF**) should (and used to) generate an integer.

### Current behaviour

Fate dice (for example, 1dF) either:
- (sidebar) It is not recognized as a valid roll (it does not matter if it is a dice tray or typed in the Dice Formula)
- (in the note as `dice: 1dF`) it is confused with what seemed to be SWRPG Narrative Dice

### Reproduction

**Sidebar**
1. Type `4dF` in the dice formula
2. Press _Roll_ button

**Note**
1. Add `dice: 4dF` on a new line
2. (optional) Click the roll icon if the automatic roll is not enabled

### Which Operating Systems are you using?

- [ ] Android
- [ ] iPhone/iPad
- [ ] Linux
- [ ] macOS
- [X] Windows

### Obsidian Version Check

1.7.4

### Plugin Version

11.4.1

### Confirmation

- [X] I have disabled all other plugins and the issue still persists.

**Comment by Calamulus** [comment]
It seems to me that the new Star Wars narrative dice overtook the "F" identifier for its Force dice rolls.

**Comment by MikhailAgudo** [comment]
Yup. I don't use Fate dice but I put `dice:3dF` out of curiosity yesterday and it gave me the Star Wars dice.

**Comment by Calamulus** [comment]
The workaround is to call the dice with a + or - modifier: using 4dF+0 works as expected for a plain Fate roll. Also works for fantasy age.
